### DISCUSSION TITLE
WBECDT1

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#835] [Q&A]
### Channel ID

WBECDT1.us

### Stream URL

https://www.becon.tv/live-stream

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19812
